### PR TITLE
Added support for minute level selection for ingestion scheduler

### LINKED ISSUE #4898: Support scheduler to run every `n` minute
**Feature**
Screenshot 2022-05-11 at 10 13 33 PM
 support above cron with ui

**Describe the task**

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Closes #4898 Added support for minute level selection for ingestion scheduler & test coverage

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image
image
image

### test coverage
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ui

### CODE REVIEW

**Comment by darth-coder00** [comment]
@ShaileshParmar11, Can you fix the CodeSmells under `[open-metadata-ui]`

**Review by darth-coder00 — COMMENTED** [review]
Can we add more tests to reach 70% branch coverage?

**Comment by ShaileshParmar11** [comment]
> Can we add more tests to reach 70% branch coverage?

I have covered all the aspects which we are using, the remaining one is due to other options which we are not using! i.e month, year, etc.

File: openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.constant.ts [hunk 1]
```diff
@@ -14,11 +14,11 @@
...
-    /* {
+    {
       label: 'minute',
       value: 'minute',
-      prep: ''
-    },*/
+      prep: '',
+    },
     {
       label: 'hour',
```
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.constant.ts:23]: Hi, we are using this for every hour and some minutes. so I have updated the minutes part!

File: openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.constant.ts [hunk 2]
```diff
@@ -14,11 +14,11 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 export const getPeriodOptions = () => {
   return [
-    /* {
+    {
       label: 'minute',
```
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.constant.ts:18]: updated

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**